tests/functional/splinter_test.c

```diff
@@ -399,8 +429,9 @@ advance_base(const test_splinter_thread_params *params,
    const uint8   lookup_positive_pct = params->lookup_positive_pct;
 
    for (uint8 spl_idx = 0; spl_idx < num_tables; spl_idx++) {
-      if (test_is_done(*done, spl_idx))
+      if (test_is_done(*done, spl_idx)) {
          continue;
+      }
```

Comment:
QS: @ajhconway , @rtjohnso - For cleanup like these ... what's our protocol? I think it's better to log a separate issue for such cleanup, so there is a record of this ask.

Then, do you wish to see a separate review cycle? Or, can I move such clean-up items as independent PRs on their own issue, w/o a review?

I want to be careful about forging ahead aggressively with "no review" PR-merges, as that may lead to lots of unsupervised check-ins, creating chaos. (Let's talk off-line, too ...)

Reply:
I'm I've two minds here. Clearly it makes review easier if these cleanups are separate PRs, but it's also convenient to just take care of this stuff as we encounter it. Let's talk offline.